index.d.ts

```diff
@@ -49,6 +49,7 @@ declare namespace Eris {
   type PossiblyUncachedGuild = Guild | Uncached;
   type PremiumTier = 0 | 1 | 2 | 3;
   type VerificationLevel = 0 | 1 | 2 | 3 | 4;
+  type GuildWidgetStyles = "shield" | "banner1" | "banner2" | "banner3" | "banner4";
```

Comment:
In TS, union type names should be singular:

```suggestion
  type GuildWidgetStyle = "shield" | "banner1" | "banner2" | "banner3" | "banner4";
```